build/dockerfiles/Dockerfile

```diff
@@ -10,39 +10,8 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-# Builder: check meta.yamls and create index.json
-FROM docker.io/node:12.19.0-alpine3.12 AS builder
-RUN apk add --no-cache py3-pip jq bash git wget skopeo && pip3 install yq jsonschema
-
-ARG LATEST_ONLY=false
-ARG USE_DIGESTS=false
-
-COPY ./build/scripts/*.sh ./build/scripts/meta.yaml.schema /build/
-COPY /.eslintrc.js /.eslintrc.js
-COPY /.git /.git
-COPY /sidecars /sidecars
-COPY /package.json /package.json
-COPY /yarn.lock /yarn.lock
-COPY ./tools /tools
-COPY /che-*.yaml /
-COPY /v3 /build/v3
-WORKDIR /build/
-
-RUN ./generate_vscode_extensions.sh
-RUN if [[ ${LATEST_ONLY} == "true" ]]; then \
-      ./keep_only_latest.sh; \
-    fi
-RUN ./generate_latest_metas.sh v3
-RUN ./check_plugins_location.sh v3
-RUN ./set_plugin_dates.sh v3
-RUN ./check_metas_schema.sh v3
-RUN if [[ ${USE_DIGESTS} == "true" ]]; then ./write_image_digests.sh v3;fi
-RUN ./index.sh v3 > /build/v3/plugins/index.json
-RUN ./list_referenced_images.sh v3 > /build/v3/external_images.txt
```

Comment:
why has this been removed? it's a useful audit tool in that you can open a plugin reg image and immediately see a simple list of image references. Has this been replaced by something else (json, yaml) ?

Reply:
@nickboldt it's there https://pr-check-815-alpine-che-plugin-registry.surge.sh/v3/external_images.txt  (just not generated by sh script)